test/utils/initMatchers.js

```diff
@@ -28,6 +28,21 @@ chai.use((chaiAPI, utils) => {
     );
   });
 
+  chai.Assertion.addMethod('toHaveVirtualFocus', function elementIsVirtuallyFocused() {
```

Comment:
Interesting, I believe we could refactor the Autocomplete's tests to use this, in case you would fancy it.

Reply:
Yeah, we should probably use is where we can. We need a better name though. `toBeActiveDescendant` is the obvious one? cc @eps1lon

Reply:
I'm trying to dissuade me to just overload `toHaveFocus` because I'm not sure we have some implicit assumptions in our existing tests using `toHaveFocus`. Though in the end we should probably just stick with the "no abstraction is better than a wrong abstraction"-thinking.

We had some discussion in the testing-library repo and my general concern with `toBeActiveDescendant` was always that I read this matcher and think "active descendant of what?". More context: https://github.com/testing-library/jest-dom/issues/207

ARIA talks about "DOM focus" regarding `document.activeElement`. Maybe `toHaveAriaFocus` to distinguish "DOM focus" and "ARIA focus"? Though that's just another way of saying `toHaveVirtualFocus`. 

I think I prefer `toHaveVirtualFocus` since it can be associated with `toHaveFocus`.

Reply:
I don’t mind `toHaveVirtualFocus` or `toHaveAriaFocus`. I’ll revert the change